src/main/groovy/com/avioconsulting/mule/linter/rule/pom/MuleRuntimeVersionRule.groovy

```diff
@@ -0,0 +1,24 @@
+package com.avioconsulting.mule.linter.rule.pom
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.RuleViolation
+
+class MuleRuntimeVersionRule extends PomPropertyValueRule {
+
+    static final String RULE_ID = 'MULE_RUNTIME'
+    static final String PROPERTY_NAME = 'app.runtime'
+    static final String RULE_NAME = 'app.runtime maven property match'
+
+    String version
```

Comment:
Why are we saving the version when it isn't used for anything after its passed to the super() constructor?

Reply:
Right not needed. Removing